web/landing/ResourcePage/FaqCard.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import Typography from "@material-ui/core/Typography";
+import { Card } from "@material-ui/core";
+import Divider from "@material-ui/core/Divider";
+import { ChevronRight } from "@material-ui/icons";
+import { LinkButton } from "../../common/LinkButton";
+import Box from "@material-ui/core/Box";
+
+const useStyles = makeStyles(theme => ({
+  faqCard: {
+    borderRadius: 10,
+    padding: theme.spacing(2),
+    backgroundColor: theme.palette.grey[100],
+    borderColor: theme.palette.primary.main,
+    height: "100%"
+  },
+  faqCardAnswer: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1)
+  },
+  faqCardDivider: {
+    marginBottom: theme.spacing(1)
+  },
+  linkWholeCard: {
```

Comment:
```suggestion
  linkWholeCard: {
    padding: 0,
```

Pour désactiver le `padding` par défaut du `LinkButton`de material. Sinon ça fait un alignement pas très propre entre les `FaqCard` et les autres items (titres, boutons, ...).

Reply:
yes, bien vu :+1: